addOns/ascanrules/src/main/javahelp/org/zaproxy/zap/extension/ascanrules/resources/help/contents/ascanrules.html

```diff
@@ -460,7 +460,10 @@ <H2 id="id-90021">XPath Injection</H2>
 to. He may even be able to elevate his privileges on the web site if the XML data is being used for 
 authentication (such as an XML based user file) or authorization." This rule attempts to identify such 
 weaknesses.
+<p><strong>Note:</strong> If the Custom Payloads addon is installed you can add your own error strings (payloads) in the Custom Payloads options panel. 
```

Comment:
I should fix this occurrence of “addon” (vs “add-on”), but may as well do it in another PR throughout the repo.

Reply:
I don't get why it shouldn't be fixed already.

Reply:
I just figured it made more sense to do them all at once.

I'm not set either way. I'll adjust this one.